lib/puppet/provider/iis_application_pool/webadministration.rb

```diff
@@ -12,10 +12,7 @@
   mk_resource_methods
 
   def exists?
-    inst_cmd = "Get-WebConfiguration -Filter '/system.applicationHost/applicationPools/add' | ? { $_.Name -eq '#{@resource[:name]}' }"
```

Comment:
Pretty sure not making the query at all will be faster than making a fast query, but I can run this too...